pkg/model/config/config.go

```diff
@@ -125,6 +130,43 @@ func (c Config) HasGroup(group string) bool {
 	return false
 }
 
+const (
+	CRDResourceName      = "customresourcedefinitions.apiextensions.k8s.io"
+	VWebhookResourceName = "validatingwebhookconfigurations.admissionregistration.k8s.io"
+	MWebhookResourceName = "mutatingwebhookconfigurations.admissionregistration.k8s.io"
+)
+
+func (c *Config) AddCoreResourceVersions(name string, versions []string) {
```

Comment:
Could we comment all exported funcs?

Reply:
Yeah I will. There are a lot of doc updates needed before this gets merged anyway, and I'm waiting on more input on the `coreResourceVersion` approach to storing CRD/webhook versions before adding godocs.